### PR TITLE
BIP0039: Introduce WLIPS (Word Lists Improvement Proposals) repository.

### PR DESCRIPTION
**Pull request by p2w34**

### CODE REVIEW

File: bip-0039.mediawiki [hunk 1]
```diff
@@ -6,7 +6,7 @@
           Pavol Rusnak <stick@satoshilabs.com>
           Aaron Voisine <voisine@gmail.com>
           Sean Bowe <ewillbefull@gmail.com>
-  Comments-Summary: Unanimously Discourage for implementation
+  Comments-Summary: The creation of word lists moved to a separate repo: https://github.com/p2w34/wlips
```
Comment (luke-jr) [inline bip-0039.mediawiki:9]: This doesn't belong here.

If you wish to leave a comment, put it at the Comments-URI wiki page. (But that's for review of this BIP, not changes proposed to it.)

File: bip-0039.mediawiki [hunk 2]
```diff
@@ -111,9 +90,17 @@ The described method also provides plausible deniability, because every passphra
 generates a valid seed (and thus a deterministic wallet) but only the correct one
 will make the desired wallet available.
 
-==Wordlists==
+==Word lists==
+
+[[https://github.com/p2w34/wlips|Creation of word lists moved to a separate repo]].
+There are multiple reasons behind it. The most important ones are:
+* word lists are not Bitcoin-specific
```
Comment (luke-jr) [inline bip-0039.mediawiki:97]: They are used by Bitcoin, so this doesn't make sense.

Altcoins copying Bitcoin isn't a reason to exclude something from BIPs.

File: bip-0039.mediawiki (hunk 1 above)
Comment (p2w34) [reply bip-0039.mediawiki:9]: I see. I revert the change to this line.

File: bip-0039.mediawiki (hunk 2 above)
Comment (p2w34) [reply bip-0039.mediawiki:97]: I remove this argument

**Comment by prusnak** [comment]
NAK on removing the `Wordlist` section. I was never a fan of localized wordlists. This fragmentation makes the situation worse. I think we should about keeping BIP39 in it's original form (maybe even reducing it to English wordlist) and if someone wants to create localized wordlists they should apply for a new BIP.

**Comment by p2w34** [comment]
It would be great to set the direction. Peoples' energy could be redirected to tackle the problem holistically, instead of being wasted on PRs which have no chances of being accepted. Hence this PR with the idea of pausing BIP0039 and suggesting to move work on word lists to a separate repo.

**Comment by luke-jr** [comment]
@p2w34 Sounds like you will need to make a new BIP for localized mnemonic codes. First step is to email the bitcoin-dev mailing list with the proposal and discuss it. Probably a good idea to get some wallets onboard.

Once there's a new BIP, perhaps @prusnak et al will consider a PR reducing this BIP to just English.

**Comment by p2w34** [comment]
@luke-jr, @prusnak I see. Let me then first do some progress on **wlips** by myself and then actually decide what to do. Thank you.

**Pull request closed without merging**